### PR TITLE
[QHC-823] Release 0.28.0

### PR DESCRIPTION
**Pull request by jjmartinezQT**
This PR brings the changes in the changelog and version number for the new release 0.28.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,307 @@
...
+
+- `platform.execute_experiment()` and the underlying `ExperimentExecutor` can now handle experiments with multiple qprograms and multiple measurements. Parallel loops are also supported in both experiment and qprogram. The structure of the HDF5 results file as well as the functionality of `ExperimentResults` class have been changed accordingly. [#796](https://github.com/qilimanjaro-tech/qililab/pull/796)
+
+- Added pulse distorsions in `execute_qprogram` for QBlox in a similar methodology to the distorsions implemented in pulse circuits. The runcard needs to contain the same structure for distorsions as the runcards for circuits and the code will modify the waveforms after compilation (inside `platform.execute_qprogram`).
+
+  Example (for Qblox)
+
+  ```json
```
Comment (fedonman) [inline CHANGELOG.md:198]: here too

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,307 @@
...
+  # Or finally, Routing with a concrete Routing instance:
+  router = Sabre(connectivity=None, lookahead=1) # No connectivity needed, since it will be overwritten by the specified in the Transpiler:
+  routed_circ, final_layouts = transpiler.route_circuit([c], placer=Trivial, router=router, coupling_map=<connectivity_to_use>)
+  ```
+
+[#821](https://github.com/qilimanjaro-tech/qililab/pull/821)
+
+- Added a timeout inside quantum machines to control the `wait_for_all_values` function. The timeout is controlled through the runcard as shown in the example:
+
+  ```json
```
[the remark this answers is not in the document]
Comment (jjmartinezQT) [reply CHANGELOG.md]: Addressed.

File: CHANGELOG.md (hunk 1 above)
Comment (jjmartinezQT) [reply CHANGELOG.md]: Addressed.

**Review by fedonman — APPROVED** [review]
🚀 🪐

**Pull request merged**